pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -81,6 +90,43 @@ def get_object(self):
             queryset, namespace=self.kwargs["namespace"], name=self.kwargs["name"]
         )
 
+    @swagger_auto_schema(
+        operation_description="Create an artifact and trigger an asynchronous task to create "
+        "Collection content from it.",
+        operation_summary="Upload a collection",
+        request_body=CollectionOneShotSerializer,
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    def create(self, request, path=None):
```

Comment:
@cutwater do you want me to move this to it's own view?

Reply:
Yes, please

Reply:
fixed in most recent push.